frontend/apps/reader/modules/readergesture.lua

```diff
@@ -686,6 +688,8 @@ function ReaderGesture:buildMenu(ges, default)
         {"suspend", true},
         {"exit", true},
         {"restart", not Device:isAndroid()},
+        {"poweroff", Device:isCervantes() or Device:isKobo() or Device:isSonyPRSTUX()},
```

Comment:
With this being duplicated twice here and also in https://github.com/koreader/koreader/blob/master/frontend/ui/elements/common_info_menu_table.lua#L76 maybe it should be abstracted into a Device:canPowerOff() or some such?

Reply:
Good tip. Thanks.